symbols/arm9.yml

```diff
@@ -1450,6 +1450,32 @@ arm9:
         r0: pointer to item to initialize
         r1: item ID
         r2: sticky flag
+    - name: InitBulkItem
+      address:
+        NA: 0x200CF78
+      description: |-
+        Initialize a struct bulk_item with the given information.
+        
+        This will resolve the quantity based on the item type. For Poké, the quantity code will always be set to 1. For thrown items, the quantity code will be randomly generated on the range of valid quantities for that item type. For non-stackable items, the quantity code will always be set to 0.
+        
+        r0: pointer to bulk item to initialize
+        r1: item ID
+    - name: BulkItemToItem
+      address:
+        NA: 0x200CFF0
+      description: |-
+        Convert a bulk_item into an equivalent item.
+        
+        r0: pointer to item to initialize
+        r1: pointer to bulk_item
+    - name: ItemToBulkItem
+      address:
+        NA: 0x200D0A0
+      description: |-
+        Convert an item into an equivalent bulk_item.
+        
+        r0: pointer to item to initialize
+        r1: pointer to bulk_item
```

Comment:
One of these doesn't match the parameter order defined in the headers. Not sure which one is wrong.